### ISSUE TITLE
The program language is broken

### ISSUE THREAD

**Issue by javatesterproject** [gluonhq/scenebuilder#798]
Good afternoon. I have a problem with Scene Builder v 24.0.0 When I start Scene Builder and when I start any fxml file, the language in Scene buider breaks, so I can't work. I have tried reinstalling Scene Buider, updating, reinstalling fonts in Windows, changing the language of the system. Here are screenshots of my problem, please help me.
![image](https://github.com/user-attachments/assets/21f46788-c278-4a94-9b59-7cac38c16304)
![image1](https://github.com/user-attachments/assets/e56ad71c-9521-42be-a7b3-908726e02bc6)

**Comment by Oliver-Loeffler** [comment]
Hi, thanks for the issue. Does this also happen using other JavaFX applications?

- Oliver

**Comment by koppor** [comment]
> Hi, thanks for the issue. Does this also happen using other JavaFX applications?

@Oliver-Loeffler yes - https://github.com/JabRef/jabref/issues/11755 - and we don't have a clue how to reproduce.

**Comment by Oliver-Loeffler** [comment]
Thanks for sharing the issue with PDFsam as well. As PDFsam makes use of JavaFX as well, it indicates that this is likely to be an issue with OpenJFX. Can you please provide some details about your platform?

- Which windows version did you use?
- Do you have other Java, JavaFX binaries/libraries on your path?
- Which locale settings are you using (System locale, User Locale, Custom Locale)?
- Which graphics driver are you using?

It would be great, if you could provide some of the mentioned details.

@abhinayagarwal 
From your point of you, are there other things to be considered?

Thanks!

**Comment by koppor** [comment]
"Issue" on Reddit: https://www.reddit.com/r/JavaFX/comments/1hhu0d5/bugs_with_scene_builder_v_2400/
